### DISCUSSION TITLE
Non english update listed for 2.18.1-2?

### DISCUSSION THREAD

**Discussion by mrjohnpoz** [bb-qq/r8152#438] [Q&A] [answered]
So my nas is showing there is an 2.18.1-2 update.  But not english? 

![update](https://github.com/user-attachments/assets/97dec021-9f7f-4a5e-9e0e-8d922cdacd63)

I have always grabbed new version from here

https://github.com/bb-qq/r8152/releases

Which only shows 2.18.1-1, is this repository just behind?

**Comment by Sjekke** [comment]
I never had a pushed upgrade, so I won't upgrade if I was you. I also don't have that logo.

**Comment by bb-qq** [comment]
Where did you download the spk package from? The package name and description seem different from the one I made.

**Comment by mrjohnpoz** [comment]
I always download from

https://github.com/bb-qq/r8152

I keep all my old installs.. Yeah I wasn't going to install it..  Not sure why its showing that.. 

Here is the hash from the latest 2.18.1-1 I downloaded from your site

CRC32: 2398C9FE
MD5: EA789738008CBED1C8398B131075EC41
SHA-1: 45D167904EDF80A987D4E6F809D8B521D7C737A9
SHA-256: A5DB1A7A7527A1D06D2400B4B1C546A2758ECE0D38DE9C585422BF32A722C35D

![files](https://github.com/user-attachments/assets/3ebeb662-9da3-444a-8dc4-94926d3452ff)

My DSM said I had a package to update - but it doesn't look like yours.. 

Ok - I had community https://synocommunity.com/ setup, when I removed that it went back to looking normal and the update went away

![2024-10-05_071204](https://github.com/user-attachments/assets/907cf656-5666-430a-9192-b6384ab887a3)

Manually look at what packages they have via web doesn't show that one.. And I have added the community back but its not showing the update to yours now.. I will see if it comes back when DSM checks if there is updates available

**Reply by bb-qq** [reply, marked as answer]
I found this package here.
https://spk7.imnks.com/?arch=geminilake

The package appears to have been created by someone on their own.
Because DSM 7 has no signature mechanism and the driver works with root privileges, I recommend not using such a package.

**Comment by mrjohnpoz** [comment]
I had no plans on doing so ;) hehehe Was just curious why was showing as an update to your driver? Thanks!

Something odd that it would think that driver is an update to yours.. Wonder how many others that maybe have auto update on or just click when they see an update might be updating to this package?

**Reply by bb-qq** [reply]
Perhaps because you have added the site's package catalog as a community package yourself.
I have added a warning about this package on the home page.

**Reply by mrjohnpoz** [reply]
nope I have never added their package from community - have always added package direct from your site. But yeah dsm was seeing the update from their community as an update. to your packaqe.

But yes I had community packages availability enabled.

**Reply by bb-qq** [reply]
What is registered in your package catalog?

It can be referenced in the “To add a package resource provided by a third party community” procedure in the manual below.
https://kb.synology.com/en-af/DSM/help/DSM/PkgManApp/configure?version=7

**Reply by mrjohnpoz** [reply]
I did have, and still do have https://packages.synocommunity.com, but have never installed from there - for this package I always do a manual download from your site.. Which is why it was weird it was showing an update to your package from different source. 

I went over all of this earlier in the thread.

**Reply by bb-qq** [reply]
Perhaps because the package identifiers were the same, the DSM recognized them as the same package even though the package sources were different. I think this limitation is unavoidable.

I can't find r8152 in SynoCommunity, so I can't fully explain what happened, but it seems difficult to investigate further.

Can we close this thread?

**Reply by mrjohnpoz** [reply]
yeah I got my answer long time ago thanks! Not your fault if some other package repository is doing such nonsense.. Glad you put up a warning about it though.

Might keep others running dsm from mistakenly updating to a version that is not from you.

**Reply by bb-qq** [reply]
Thanks for the reply. Now I will close this thread.

**Comment by gaoo90** [comment]
This is a fake update  from https://t.me/spkimnks ? (https://imnks.com/1780.html)

![image](https://github.com/user-attachments/assets/d074d7ab-f3f1-4fee-b221-d3f3d7e2fab6)

![image](https://github.com/user-attachments/assets/4f2eb321-4b80-4c19-b88f-44529d8149b3)
